docs/rules/reuse-random-objects.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Reuse Random Objects
+ruleId: ReuseRandomObjects
+since: 3.20.0
+minJavaVersion: 1.1
+remediationCost: 5
+links:
+    - displayName: "S2119"
+      url: "https://sonarcloud.io/organizations/default/rules?languages=java&open=java%3AS2119&q=S2119"
+    
+description:
+    .
+tags: ["Java 1.1", "Security"]
+---
+
+# Reuse Random Objects
+
+[[toc]]
+
+## Properties
+
+<RuleProperties />
+
+## Description
+
+Creating a new Random object each time a random value is needed is inefficient and may produce numbers which are not random. This rule extracts reusable [java.util.Random](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html) objects, from local variables to class or instance fields.
+
+## Benefits
+
+Improves the unpredictability and efficiency of the generated random values.
+
+## Code Changes
```

Comment:
Suggestion: Use one example where the `new SecureRandom()` constructor is used instead of `new Random()` to highlight that both cases get transformed.